packages/crypto/src/webcrypto-browser.ts

```diff
@@ -6,7 +6,7 @@ export default {
   get (win = globalThis) {
     const nativeCrypto = win.crypto
 
-    if (nativeCrypto == null || nativeCrypto.subtle == null) {
+    if (nativeCrypto?.subtle == null) {
```

Comment:
If crypto is not supported at all (which would be extremely rare in modern environments), `win.crypto` would be `undefined`  so isn't `if (nativeCrypto == null || nativeCrypto.subtle == null)` safer?

Reply:
Optional chaining makes the additional null guard unnecessary:

```js
const derp = null
// null

derp?.foo
// undefined

derp == null
// true

derp?.foo == null
// true
```